test/system/flows/mentor_changes_mentored_tracks_test.rb

```diff
@@ -19,8 +19,8 @@ class MentorChangesMentoredTracksTest < ApplicationSystemTestCase
         visit mentoring_queue_path
         within(".mentor-queue-filtering") { click_on "Ruby" }
         click_on "Change the tracks you mentor"
-        find("label.track", text: "Ruby").click
-        find("label.track", text: "C#").click
+        find("label.track", text: "Ruby\nRecent median waiting time: ~16 minutes").click
+        find("label.track", text: "C#\nRecent median waiting time: ~33 minutes").click
```

Comment:
`Recent median waiting time` needs to change to `Avg. wait time` as it won't fit responsively here if it's that long. I don't mind it changing in `common` to this too.

Reply:
What is the limit for fitting responsively? Do we have room here for `Average wait time`?
(Fewer abbreviations means less confusion, particularly to non-English speakers unfamiliar with this abbreviation.)

Reply:
I'm not sure. If you add an issue referencing this to exercism/exercism we'll take a look in due course (otherwise I'll forget a comment in an old PR here)